listener/utils/fake_logger.py

```diff
@@ -0,0 +1,85 @@
+"""
```

Comment:
We could also use loremipsum.get_sentece package which generates a random lorem ipsum sentence.